backend/kernelCI_app/views/treeDetailsBuildsView.py

```diff
@@ -65,23 +69,20 @@ def _sanitize_rows(self, rows):
             ):
                 self._process_builds(row_data)
 
-    @extend_schema(
-        parameters=[
-            COMMIT_HASH_PATH_PARAM,
-            TreeQueryParameters,
-        ],
-        responses=TreeDetailsBuildsResponse,
-        methods=["GET"],
-    )
-    def get(self, request, commit_hash: str | None) -> Response:
-        origin_param = request.GET.get("origin")
-        git_url_param = request.GET.get("git_url")
-        git_branch_param = request.GET.get("git_branch")
-
+    def get(
+        self,
+        request: HttpRequest,
+        git_url: Optional[str],
+        tree_name: Optional[str],
+        git_branch: Optional[str],
```

Comment:
This way you don't need to instantiate this parameters on the `super().get(...)`. You will need to reorganize the params to put all the default arguments at the end.
```suggestion
        git_url: Optional[str] = None,
        tree_name: Optional[str] = None,
        git_branch: Optional[str] = None,
```

Reply:
I wanted to leave them without defaults in order to "force" the validation in the children and make it explicit when one of the fields is always expected to be None or not